tables/DataMan/test/tDataManInfo.cc

```diff
@@ -0,0 +1,228 @@
+//# tDataManInfo.cc: Test program for class DataManInfo
+//# Copyright (C) 2021
```

Comment:
2022? ;-)

Reply:
No, it has been created in 2021. I could add ,2022 but that is not necessary.